README.Rmd

```diff
@@ -37,11 +38,12 @@ Click on the Addins drop-down and select **Start runbisonpic** App.
 
 ### User Guide
 
-To access the user guide go to XXXX (insert link).
+To access the user guide click [here] (https://poissonconsulting.github.io/bisonpicsuite/bisonpic-user-guide.html).
```

Comment:
Same comment as in shinybisonpic pull request review regarding how to phrase hyperlinks with accessibility in mind.